docs/operators/ftpInputOperator.md

```diff
@@ -0,0 +1,72 @@
+FTP Input Operator
+=============
+
+## Operator Objective
+This operator(`AbstractFTPInputOperator`) is designed to scan a directory from an FTP server for files, read
+and split file content into tuples such as lines or blocks of bytes, and finally
+emit them on the output port for further processing by downstream operators.
+The operator extends the `AbstractFileInputOperator`. It overrides the
+getFSInstance() method and returns an instance of the FTPFileSystem
+(`org.apache.hadoop.fs.ftp.FTPFileSystem`)
+
+## Class Diagram
+![FTPInputOperator class diagram](images/ftpInputOperator/classdiagram.png)
+
+## Operator Information
+1. Operator location : **_malhar-lib_**
+2. Available since : **_2.0.0_**
+3. Java Package : [com.datatorrent.lib.io](https://github.com/apache/apex-malhar/blob/master/library/src/main/java/com/datatorrent/lib/io/AbstractFTPInputOperator.java)
+
+### Ports
+Because this is an input operator, there are no input ports.
+
+
+| Port  | Description | Type | Mandatory |
+| -------  | -------- | ----- | ------- |
+| *output* | output port on which data is emitted | String | Yes |
+
+
+## Partitioning
+#### Static Partitioning
+Configure parameter `partitionCount` to define the desired number of initial partitions
+(4 in this example).
+
+```xml
+<property>
+  <name>dt.operator.{OperatorName}.prop.partitionCount</name>
+  <value>4</value>
+</property>
+```
+Alternatively, this can be changed in the application code by setting the operator property `partitionCount` to the desired number of partitions:
+
+```java
+FTPStringInputOperator reader = dag.addOperator("Reader", new FTPStringInputOperator());
+reader.setPartitionCount(4);
+```
+
+#### Dynamic Partitioning
+Dynamic partitioning -- changing the number of partitions of one or more operators
+in a running application -- can be achieved in multiple ways:
```

Comment:
This is done via the setPartitionCount method which is a part of the super class AbstractFileInputOperator.